src/tests/Common/testenvironment.proj

```diff
@@ -37,6 +37,7 @@
       DOTNET_EnableSSE41;
       DOTNET_EnableSSE42;
       DOTNET_EnableSSSE3;
+      DOTNET_EnableAPX;
```

Comment:
this is supposed to get used below in one of the pipeline. Is it intended to not do so at this point? If yes, then may be just remove it and add back when we enable the pipeline to test it?

Reply:
This I was not sure if needed. I added it since we provide a flag to turn it off for other similar features. I can remove it

Reply:
yes, can you please remove it unless we want to add pipeline for it?